packages/core/src/lib/core/context/request-context.ts

```
@@ -244,6 +244,20 @@ export class RequestContext {
 		return null;
 	}
 
+	/**
+	 * Retrieves the current integration tenant from the request context.
+	 *
+	 * @param {boolean} throwError - Flag indicating whether to throw an error if integration tenant is not found.
+	 * @returns The current integration tenant if found, otherwise null.
+	 */
+		static currentIntegrationTenant(): IIntegrationTenant | null {
+			const requestContext = RequestContext.currentRequestContext();
+			if (requestContext && requestContext._req) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)